packages/components/src/Hidden/Hidden.stories.mdx

```diff
@@ -0,0 +1,64 @@
+import { Meta, Story, Preview } from '@storybook/addon-docs/blocks';
+import { Button, Hidden, Text } from '@marigold/components';
+import { useState } from 'react';
+
+<Meta title="Components/Layout/Hidden" />
+
+# Hidden
+
+## Description
+
+With the Hidden component you can hide elements or components. E.g. to hide some text in specific cases, for groups or platforms.
+The element uses always the themeSection with the name: `layout` in your given theme. The variant in your section can be added with the variant prop. The default variant is `hidden`.
```

Comment:
There is no "hidden" variant in the layout themeSection so far, right? Would make sense to add an empty object there maybe